src/test/operators/difference_test.cpp

```diff
@@ -79,4 +80,32 @@ TEST_F(OperatorsDifferenceTest, ThrowWrongColumnOrderException) {
   EXPECT_THROW(difference->execute(), std::exception);
 }
 
+TEST_F(OperatorsDifferenceTest, ForwardOrderByFlag) {
+  // Verify that order_by flag is not set when not present in left input
```

Comment:
Seeing that I have the same comments here in `difference_test.cpp` as well as in `projection_test.cpp` and in `table_scan_test.cpp` (and partially overlapping with `validate_test.cpp` as well): Have you looked into factoring out a (properly templated by the operator class or something like that) util function that could be called here and in the other named tests instead of having similar code fragments spread across the files? I haven't checked the exact difference between those methods yet, though – but only judging from their general structures and the appearance of the exact same comments, it seems like there could be some to-be-factored-out overlap. 🙂